src/test/common/TestdataGenerator.kt

```diff
@@ -572,6 +580,38 @@ fun lagPersonResultat(
     return personResultat
 }
 
+fun lagTilkjentYtelse(utbetalingsoppdrag: Utbetalingsoppdrag, behandling: Behandling) =
+    TilkjentYtelse(
+        utbetalingsoppdrag = objectMapper.writeValueAsString(utbetalingsoppdrag),
+        behandling = behandling,
+        opprettetDato = LocalDate.now(),
+        endretDato = LocalDate.now()
+    )
+
+fun lagUtbetalingsoppdrag(utbetalingsperiode: List<Utbetalingsperiode>) = Utbetalingsoppdrag(
+    kodeEndring = Utbetalingsoppdrag.KodeEndring.NY,
+    fagSystem = "BA",
+    saksnummer = "",
+    aktoer = UUID.randomUUID().toString(),
+    saksbehandlerId = "",
+    avstemmingTidspunkt = LocalDateTime.now(),
+    utbetalingsperiode = utbetalingsperiode
+)
+
+fun lagUtbetalingsperiode(opphør: Opphør? = null) = Utbetalingsperiode(
+    erEndringPåEksisterendePeriode = false,
+    opphør = opphør,
+    periodeId = 0,
+    datoForVedtak = LocalDate.now(),
+    klassifisering = "BATR",
```

Comment:
```suggestion
    klassifisering = "KS",
```
Samme her siden BATR står for barnetrygd